docs/guides/event_loop.md

```diff
@@ -2,14 +2,76 @@
 
 # An Introduction to the Event Loop in napari
 
+
+## tldr;
+
+It is not necessary to have a deep understanding of Qt or event loops to use
+napari. napari attempts to use "sane defaults" for most scenarios. Here are the
+most important details:
+
+### In IPython or Jupyter Notebook
+
+napari will detect if you are running an an IPython or Jupyter shell, and will
+automatically use the [IPython GUI event
+loop](https://ipython.readthedocs.io/en/stable/config/eventloops.html#integrating-with-gui-event-loops).
+As of [version 0.4.7](https://github.com/napari/napari/releases/tag/v0.4.7) is
```

Comment:
I'd prefer the docs focus on what we do now, and de-emphasize what we used to do.

It's important to give people the info that we no longer need to call `%gui qt`, but this is probably better distributed in the release notes, on zulip, and twitter, etc.

Reply:
I did try to deemphasize it with the offset, but given the 2 year usage... I kinda thought a lot of people would be asking "what about gui_qt??"  So it'd be nice to send them somewhere to answer that

Reply:
(Also not needing `%gui qt` magic is a different point than this one, which is about `napari.gui_qt`)

Reply:
I think this is ok for a transition period, but maybe in 3-6 months we can revisit and update the language a bit

Reply:
yeah, definitely doesn't have to be there forever